scripts/eve.coffee

```diff
@@ -70,27 +72,52 @@ playersOnline = (msg) ->
 lookupSystemKills = (msg) ->
 	name = msg.match[1].toLowerCase()
 	ops = []
-	
+
 	ops.push (cb) ->
 		client.hget 'solarsystems_name_to_id', name, cb
-	
+
 	ops.push (id, cb) ->
 		return msg.send("Either #{name.capitalize()} has had no kills in the last hour, or that system doesn't exist.") if not id
 		client.hget 'kills_by_id', id, cb
-	
+
 	ops.push (data, cb) ->
 		data = JSON.parse(data)
 		msg.send "#{name.capitalize()} has had #{pnum(data.shipKills)} ship kills, #{pnum(data.factionKills)} faction kills, and #{pnum(data.podKills)} pod kills in the last hour."
 		cb()
-	
+
 	async.waterfall ops
 
+lookupTheraDistance = (msg) ->
+	system = msg.match[1].toLowerCase()
+
+	request.get "#{evescout}/wormholes?systemSearch=#{system}", (err, resp) ->
+		return console.log(err) if err
+		data = JSON.parse(resp.body)
+
+		# I daren't make this recursive, it does not sound like a good idea. But I want to.
+		jumps = 0
+		try
```

Comment:
Try works, but is not preferred as this is more python (and erlang, kinda) style. Check instead for what the body returns on an error/invalid system.